NewForce-Cohort-9/tabloidmvc-josh-jacobs-jingleheimer-schmidt
# Ticket 8-Edit Post for cbbranch-1

# Description

Added feature to edit a post. You can now successfully edit a post that is saved to the database. [Trello Ticket #8](https://trello.com/c/ljROqFhX)

## Type of change


# Testing Instructions

1. `git add  .` and  `git commit -m ""`  whatever you're working on on your branch
2.  `git checkout main`
3.  `git fetch --all`
4.  `git checkout cbbranch-1`
5. If you don't see the most recent version of my files, `git pull origin cbbranch-1`
6. If you haven't created the Tabloid MVC database, please do so.
7. Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-6/tabloidmvc-the-bit-o-honeys/blob/main/SQL/01_TabloidMVC_Create_DB.sql)
8. Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-6/tabloidmvc-the-tootsie-rolls/blob/main/SQL/02_TabloidMVC_Seed_Data.sql)
9. Click the execute with debugger solid green button to launch the app
10. If not logged in, please go to Login on the nav bar and log in with this email: admin@example.com
11. Click on "Posts" and you will redirected to the /Post page.
12. You will see a list of posts with their title, author, category, published displayed.
13. Locate the one with a title of "Cute Cow". Click on the blue pencil icon in the middle, next to the published date, you will be directed to /Post/Edit/2.
14.  Click in the field for "title" and update the title to saw "New Cute Cow".
15. Click the drop down on the "Category" field and change *Entertainment* to *History*.
16. Click the blue button: "Save". You will be brought back to your list of posts and see you changes you made.
17. Click on the blue pencil button to edit the post titled "howdy".
18. Click on the title field and change it to say "hello".
19. Click on the red "Cancel" button. You will be brough back to the Post List Index page and you should not see your changes.



# Checklist:
